utility/rp_client.py

```diff
@@ -164,10 +165,11 @@ def process(xmlObj, rportal):
             process_testcase(xmlObj, testcase, tsuite)
         log.info("\nFinished testcases")
         fqpath = os.path.join(xmlObj._configs.payload_dir, "attachments")
-        if os.path.exists(f"{fqpath}/{tsuite.xml_name}/{tsuite.xml_name}.zip"):
-            rplog.add_attachment(
-                tsuite.item_id, f"{fqpath}/{tsuite.xml_name}/{tsuite.xml_name}.zip"
-            )
+        log_zip = glob.glob(
+            os.path.join(f"{fqpath}/{tsuite.xml_name}/", f"{tsuite.xml_name}.*")
+        )
+        if log_zip:
+            rplog.add_attachment(tsuite.item_id, f"{log_zip[0]}")
```

Comment:
Let's specifically look for `tar.gz`. What do you think?

Reply:
As we are only archiving with tar.gz.
It is good to check with a particular extension.
I modified the tar.gz extenstion